package.json

```diff
@@ -22,7 +22,8 @@
     "modal"
   ],
   "dependencies": {
-    "react-native-vector-icons": "^4.0.0"
+    "prop-types": "^15.5.10",
+    "react-native-vector-icons": "^4.2.0"
```

Comment:
Hey @iRoachie could you please update react-native-vector-icons in a different commit? 🤓